src/drivermodels/IDMDriver.jl

```diff
@@ -43,13 +43,13 @@ mutable struct IDMDriver <: LaneFollowingDriver
         retval
     end
 end
-get_name(::IDMDriver) = "IDM"
+
 function set_desired_speed!(model::IDMDriver, v_des::Float64)
     model.v_des = v_des
     model
 end
 
-function AutomotiveDrivingModels.observe!(model::IDMDriver, scene::Frame{Entity{VehicleState, BicycleModel, Int}}, roadway::Roadway, egoid::Int)
+function AutomotiveDrivingModels.observe!(model::IDMDriver, scene::Scene{Entity{VehicleState, BicycleModel, Int}}, roadway::Roadway, egoid::Int)
```

Comment:
This will break with ADM v0.7, it should work with v0.8
since there is no testing for those file let's bother about this later though.